modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapImage.vue

```diff
@@ -0,0 +1,256 @@
+<template>
+  <div class="apos-image-control">
+    <AposButton
+      type="rich-text"
+      @click="click"
+      :class="{ 'apos-is-active': buttonActive }"
+      :label="tool.label"
+      :icon-only="!!tool.icon"
+      :icon="tool.icon ? tool.icon : false"
+      :modifiers="['no-border', 'no-motion']"
+    />
+    <div
+      v-if="active"
+      v-click-outside-element="close"
+      class="apos-popover apos-image-control__dialog"
+      x-placement="bottom"
+      :class="{
+        'apos-is-triggered': active,
+        'apos-has-selection': true
+      }"
+    >
+      <AposContextMenuDialog
+        menu-placement="bottom-start"
+      >
+        <AposSchema
+          :schema="schema"
+          :trigger-validation="triggerValidation"
+          v-model="docFields"
+          :utility-rail="false"
+          :modifiers="formModifiers"
+          :key="lastSelectionTime"
+          :generation="generation"
+          :following-values="followingValues()"
+          :conditional-fields="conditionalFields()"
+        />
+        <footer class="apos-image-control__footer">
+          <AposButton
+            type="default" label="apostrophe:cancel"
+            @click="close"
+            :modifiers="formModifiers"
+          />
+          <AposButton
+            type="primary" label="apostrophe:save"
+            @click="save"
+            :modifiers="formModifiers"
+          />
+        </footer>
+      </AposContextMenuDialog>
+    </div>
+  </div>
+</template>
+
+<script>
+import AposEditorMixin from 'Modules/@apostrophecms/modal/mixins/AposEditorMixin';
+
+export default {
+  name: 'AposTiptapImage',
+  mixins: [ AposEditorMixin ],
+  props: {
+    name: {
+      type: String,
+      required: true
+    },
+    tool: {
+      type: Object,
+      required: true
+    },
+    editor: {
+      type: Object,
+      required: true
+    }
+  },
+  data() {
+    return {
+      generation: 1,
+      triggerValidation: false,
+      docFields: {
+        data: {}
+      },
+      active: false,
+      formModifiers: [ 'small', 'margin-micro' ],
+      originalSchema: [
+        {
+          name: '_image',
+          type: 'relationship',
+          label: apos.image.label,
+          withType: '@apostrophecms/image',
+          required: true,
+          max: 1,
+          // Temporary until we fix our modals to
+          // stack interchangeably with tiptap's
+          browse: false
+        },
+        ...(getOptions().imageStyles ? [
+          {
+            name: 'style',
+            label: this.$t('apostrophe:style'),
+            type: 'select',
+            choices: getOptions().imageStyles,
+            def: getOptions().imageStyles?.[0].value
```

Comment:
default selected is `null`
 🤔 
<img width="301" alt="image" src="https://user-images.githubusercontent.com/8301962/215470785-6055457e-e82a-43be-82d2-9ab336b48294.png">